### DISCUSSION TITLE
Support for HmIP-BBL-2

### DISCUSSION THREAD

**Discussion by PetzJohannes** [SukramJ/aiohomematic#1166] [General]
Hello,

I am new to Homematic and have little experience with Home Assistant. We moved to a new apartment with pre-installed Homematic Cover actors. These actors are HmIP-BBL-2. In the source code I found the HmIP-BBL reference, but without '-2'.

So my question is, is the BBL-2 supported? Is there anything I can do to get support for the BBL-2? If someone helps or guides me, I may be able to contribute the device support.

Greeting from Germany,

Johannes

**Comment by SukramJ** [comment]
Yes, support is for HmIP-BBL*.

**Comment by PetzJohannes** [comment]
Thanks for the fast reply!

I have added 3 actors to my CCU3. But neither the Home Assistant integration icon for Homematic appears nor one of the devices in the device list. I added my configuration to the Home Assistant configuration.yaml. Do you have a clue for me, where to search or start?

 I enabled the debug log for the homematic integration and it shows a list of devices and ids.

**Reply by SukramJ** [reply]
This integration is not installed via yaml. Which installation guide did you follow?
Are you sure you read the [docs](https://github.com/danielperna84/custom_homematic#homematicip-local-documentation) for this integration?

**Reply by SukramJ** [reply]
This is the best [installation guide](https://github.com/jens-maus/RaspberryMatic/wiki/Installation-HomeAssistant#home-assistant-integration-setup).

**Reply by PetzJohannes** [reply]
I installed the homematic integration via configuration.yaml.

With this documentation: https://www.home-assistant.io/integrations/homematic/

I am running Home Assistant in a docker container. And Homematic via a CCU3.

**Reply by SukramJ** [reply]
This is the repo for the custom component and not for the HA build in integration.
I recommend you to use this integration, otherwise your devices are not supported.

**Reply by PetzJohannes** [reply]
Thank you very much!!! I was search wrong and this integration works perfectly.
